code/headers/temperature_frame_action.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <frame_action.hpp>
+#include <hwlib.hpp>
+
+namespace r2d2::robos {
+    class temperature_frame_action_c
+        : public frame_action_c<frame_type::TEMPERATURE> {
+    private:
+        // Display frame types
+        frame_cursor_color_s cursor_color;
+        frame_cursor_position_s cursor_position;
+        frame_display_8x8_character_via_cursor_s display_characters;
+
+        static constexpr char temperature_message[11] = "Obj temp: ";
+
+    public:
+        temperature_frame_action_c(base_comm_c &comm, frame_s &frame);
+
+        virtual void reply_to_data();
```

Comment:
Don't forget `override`

Reply:
fixed in https://github.com/R2D2-2019/RobOS/pull/2/commits/861ab8dbddb7383a995f5ff866feb170625e4275